jungfrau_utils/jf_file.py

```diff
@@ -79,14 +79,17 @@ def __exit__(self, *args):
         self.close()
 
     def __getitem__(self, item):
-        if isinstance(item, int):
+        if isinstance(item, int) or isinstance(item, slice):
             ind, roi = item, ()
         else:
-            ind, roi = item[0], item[1:]
+            ind, *roi = item
```

Comment:
It might be better to keep the original unpacking, where roi is a tuple and not a list. That way the new code on lines 106-109 is also not needed.

Reply:
Ah, I see. I had switched this to a list in order to append a `None:None` slice in case of `len(roi) == 1`. I didn't do that in the end. So, I am indifferent about list or tuple.

Anyway, wouldn't the `...` or `:` still be needed in 106-109?

Would the tuple work differently than the list in that case? I thought neither would work:
`jf_data[..., tuple(roi)]`
`jf_data[..., list(roi)]`
What we would need is
`jf_data[..., *roi]`
which is sadly not valid syntax.